clouddriver-openstack/src/main/resources/asg.yaml

```diff
@@ -27,10 +27,46 @@ parameters:
     description: Load balancer to use for this group
   security_groups:
     type: comma_delimited_list
-    description: security groups to use
+    description: Security groups to use
   subnet_id:
     type: string
     description: Subnet used to allocate a fixed IP for each server
+  autoscaling_type:
+    type: string
+    description: Type of autoscaling to perform. can be cpu_util, network.incoming.bytes.rate, or network.outgoing.bytes.rate
+    default: cpu_util
+  scaleup_cooldown:
+    type: number
+    description: Minimum amount of time (in seconds) between scaleup operations
+    default: 60
+  scaleup_adjustment:
+    type: number
+    description: Amount by which to change the instance count. Can be positive or negative
+    default: 1
+  scaleup_period:
+    type: number
+    description: Amount of time (in seconds) before the scaleup action is taken
+    default: 60
+  scaleup_threshold:
+    type: number
+    description: threshold that causes the scaleup action to occur, if held for scaleup_period seconds
+    default: 50
+  scaledown_cooldown:
+    type: number
+    description: Minimum amount of time (in seconds) between scaledown operations
+    default: 60
+  scaledown_adjustment:
+    type: number
+    description: Amount by which to change the instance count. Can be positive or negative
```

Comment:
Maybe it is just openstack is odd, but I just feel like this should always be positive. Scale down implies it is going down and this is just the count of how many. But this has to act how Openstack wants it.